### PR TITLE
Add pyDOE2

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: recipes/pydoe2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  version: {{ version }}
+
+source:
+    fn: pyDOE2-{{ version }}.zip
+    url: https://pypi.io/packages/source/p/pyDOE2/pyDOE2-{{ version }}.zip
+    sha256: c5c0c343bec9b7d0f6650f2d0686802c212ed36ef8b1f895d9cde2ea62768736
+
+build:
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
```
Comment (marcelotrevisani) [inline recipes/pydoe2/meta.yaml]: Please, change this part to:

```
script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
```

File: recipes/pydoe2/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    - scipy >=0.13
+    - numpy >=1.8
+
+test:
+  imports:
+    - pyDOE2
+
+about:
+  home: https://github.com/clicumu/pyDOE2
+  license: BSD 3-Clause
```
Comment (marcelotrevisani) [inline recipes/pydoe2/meta.yaml]: please, add the ``license_family`` and also the ``license_file``

File: recipes/pydoe2/meta.yaml (hunk 1 above)
Comment (jschueller) [reply recipes/pydoe2/meta.yaml]: ok

File: recipes/pydoe2/meta.yaml (hunk 2 above)
Comment (jschueller) [reply recipes/pydoe2/meta.yaml]: ok, (opened https://github.com/clicumu/pyDOE2/pull/7)

File: recipes/pydoe2/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+test:
+  imports:
+    - pyDOE2
+
+about:
+  home: https://github.com/clicumu/pyDOE2
+  license_family: BSD
+  license: BSD 3-Clause
+  # https://github.com/clicumu/pyDOE2/pull/7
+  #license_file: LICENSE
```
Comment (synapticarbors) [inline recipes/pydoe2/meta.yaml]: Can we just manually package the `license_file` for now:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

and then we can get this merged and update the feedstock when the PR gets merged after the next release.

Comment (jschueller) [reply recipes/pydoe2/meta.yaml]: ok

**Pull request merged**